eventarc/pubsub/main.go

```diff
@@ -18,33 +18,37 @@
 package main
 
 import (
-	"encoding/json"
+	"encoding/base64"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"net/http"
 	"os"
-)
 
-// PubSubMessage is the payload of a Pub/Sub event.
-// See the documentation for more details:
-// https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage
-type PubSubMessage struct {
-	Message struct {
-		Data []byte `json:"data,omitempty"`
-		ID   string `json:"id"`
-	} `json:"message"`
-	Subscription string `json:"subscription"`
-}
+	pubsub "github.com/googleapis/google-cloudevents-go/cloud/pubsub/v1"
```

Comment:
This is a surprising import path. `vN` has a special meaning in Go. It's usually only used for `v2`+. But, this is not a `v1` module -- it's `v0`.

If there is ever a `pubsub/v2`, it will probably need to be a `v2` module. But, there should only be one module at the root of the repo...